client/my-sites/upgrades/domain-management/dns/dns-add-new.jsx

```diff
@@ -155,7 +146,7 @@ const DnsAddNew = React.createClass( {
 
 		return (
 			<form className="dns__add-new">
-				<div className={ classes }>
+				<div className="dns__form-content">
```

Comment:
This class used to be `form-content`. It's OK to change it to `dns__form-content` but now there's a CSS declaration for both. If we can get rid of one of them by removing the declaration of `form-content` or changing this to `dns__form-content` it would be nicer.